SPECS/complianceascode/complianceascode.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Signatures": {
+     "complianceascode-1.0.tar.xz": "27aab4160d54ce65d4f0456e4fff04dd6fac2eed6d02a5f80620fed013b6457e"
+    }
+}
+  
```

Comment:
nit - This signatures file is formatted weirdly. Can you use `SRPM_FILE_SIGNATURE_HANDLING` to generate these signature files?

https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/docs/building/building.md#source-hashes